.github/workflows/deploy-beta.yml

```diff
@@ -97,17 +104,20 @@ jobs:
               const prRepo = context.payload.pull_request.head.repo.full_name
               const prRef = context.payload.pull_request.head.ref
 
-              refName = (prRepo === targetRepo) ? prRef : `pr${context.issue.number}`
+              refSlug = `pr${context.issue.number}`;
+              refName = (prRepo === targetRepo) ? prRef : refSlug;
               githubBranch = refName
```

Comment:
Since `githubBranch` can become to "pr123", `output.ci_commit_branch` will be assigned it as well. And then we'll refer to it as `CI_COMMIT_BRANCH` in later code, that looks a little confusing since it is an image tag that is passing, not branch. Otherwise looks like it does not break things.

Reply:
The value of CI_COMMIT_BRANCH was used only in build-and-test_dev.yml as the tag for the dev image. This PR makes CI_COMMIT_BRANCH effectively a flag to detect if workflow triggers for the branch.

Reply:
Does `CI_COMMIT_BRANCH=pr123` mean the workflow is triggered for a branch? If yes, there is no "commit branch" named "pr123". Why it is called "commit branch" then? Probably, it is "build tag" or "dev build tag".

Reply:
Got reasons to simultaneous support of Gitlab and Github flows, hence the naming weirdness. Will torerate it for the time being.